docs/user_guide/write_api/push_job.md

```diff
@@ -27,6 +28,14 @@ When performing an Incremental Push, no new dataset versions are created, and da
 of the dataset. This leverages the same mechanism as Streaming Writes, and requires that the store be configured as 
 Hybrid.
 
+### Targeted Region Push
+Technically, targeted region push is a type of full push, but it allows writing data into a subset of global regions/data
+centers, whereas full push writes globally at once. 
+
+Please note that this feature is still under active development. It doesn't push data to the rest of unspecified regions
```

Comment:
Let's use "functionality" instead of "feature" since the latter is an overloaded term in ML parlance.